projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -248,12 +249,130 @@ function ( $a, $b ) {
 						);
 					}
 
-					$return[ $key ] = $combined_users;
+					$response[ $key ] = $combined_users;
 					break;
+				// Total users from the site. Only available to users with the `list_users` capability (Super admins and admins).
+				case 'total_users':
+					if ( ! empty( $args['total_users'] ) && true === $args['total_users'] && $can_list_users ) {
+						$cache_key   = 'rest-api-list-users-' . $blog_id . '-total-count';
+						$total_users = wp_cache_get( $cache_key, 'WPCOM_JSON_API_List_Users_Endpoint' );
+						if ( false !== $total_users ) {
+							$response[ $key ] = $total_users;
+							break;
+						}
+
+						$viewer_count = 0;
+						if ( $include_viewers ) {
+							$viewer_count = $this->get_viewers_count( $blog_id );
+						}
+						$user             = count_users( 'time', $blog_id );
+						$total_users      = isset( $user['total_users'] ) ? (int) $user['total_users'] : 0;
+						$total_users      = $total_users + $viewer_count;
```

Comment:
> Plugin PhanPluginDuplicateExpressionAssignmentOperation Can simplify this assignment to
> $total_users += $viewer_count FAQ on Phan issues: pdWQjU-Jb-p2

I don't agree with this. Writing in a readable fashion should not be disallowed.